apps/api/src/app/subscribers/e2e/handle-chat-oauth.e2e.ts

```diff
@@ -0,0 +1,157 @@
+import { expect } from 'chai';
+import axios from 'axios';
+import * as sinon from 'sinon';
+
+import { UserSession } from '@novu/testing';
+import { ChannelTypeEnum, ChatProviderIdEnum } from '@novu/shared';
+import { SubscriberRepository } from '@novu/dal/';
+
+const axiosInstance = axios.create();
+
+describe('HandleCharOauth - /:subscriberId/:providerId/:environmentId (GET)', function () {
```

Comment:
```suggestion
describe('HandleChatOauth - /:subscriberId/:providerId/:environmentId (GET)', function () {
```

Reply:
Good catch this one was on all the files 😂 missed this one on the typo fixes